src/main/java/org/minions/devfund/Damian/CharCount.java

```diff
@@ -0,0 +1,22 @@
+package org.minions.devfund.damian;
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') This file "CharCount.java" should be located in "org/minions/devfund/damian" directory, not in "/home/travis/build/MinionsGroup/devfund-h/src/main/java/org/minions/devfund/Damian". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1598)

Reply:
Please rename the directory as well